### ISSUE TITLE
Test set used for Darcy experiments: "piececonst_r421_N1024_smooth2.mat"

### ISSUE THREAD

**Issue by AhmedFrikha** [devzhk/PINO#7]
Hi, 
thanks for sharing the code and the link to the training sets. Could you please also share the link to the test set used for the Darcy experiments "piececonst_r421_N1024_smooth2.mat" ? 

Thanks !!

**Comment by devzhk** [comment]
Hi, 

Thanks for being interested in our work. Here is the data for Darcy experiments. 
[piececonst_r421_N1024_smooth1.mat](https://hkzdata.s3.us-west-2.amazonaws.com/PINO/piececonst_r421_N1024_smooth1.mat)
[piececonst_r421_N1024_smooth2.mat](https://hkzdata.s3.us-west-2.amazonaws.com/PINO/piececonst_r421_N1024_smooth2.mat)

I'll add links for all other data to README as well later today. 
Best

**Comment by liroujiayou** [comment]
hello,dear my friend,[piececonst_r421_N1024_smooth1.mat](https://hkzdata.s3.us-west-2.amazonaws.com/PINO/piececonst_r421_N1024_smooth1.mat)is did'nt found.Can you give me again.Thanks！
